spec/jobs/release_object_job_spec.rb

```diff
@@ -126,15 +126,15 @@
         subject.perform(bulk_action_no_process_callback.id, params)
         expect(bulk_action_no_process_callback.druid_count_total).to eq pids.length
       end
-      # no information leakage to an unauthorized user
       it 'logs no druid info to logfile' do
```

Comment:
Does it make more sense for this to say "logs druid info to logfile" with the most recent change?